src/test/java/javaslang/collection/AbstractTraversableTest.java

```diff
@@ -1377,7 +1378,7 @@ public void shouldWriteToPrintWriter() {
         final StringWriter sw = new StringWriter();
         final PrintWriter out = new PrintWriter(sw);
         ofAll(1, 2, 3).out(out);
-        assertThat(sw.toString()).isEqualTo(ofAll(1, 2, 3).mkString("", "\n", "\n"));
+        assertThat(sw.toString()).isEqualTo(ofAll(1, 2, 3).mkString("", lineSeparator(), lineSeparator()));
```

Comment:
Great, I think that solves a problem we recently had (without any solution).

I ask myself, if it really should be

```java
mkString("", lineSeparator(), lineSeparator())
```

because we changed the argument order of `mkString` from `infix, prefix, suffix` to `prefix, infix, suffix`.

Therefore the following may be right:

```java
mkString(lineSeparator(), "", lineSeparator())
```

But if the tests pass, it probably is right as-is.